modules/profile/manifests/jenkins.pp

```diff
@@ -1,4 +1,38 @@
 class profile::jenkins::master {
-  include java8
+  $dports          = ['8080']
+  $log_proto       = 'tcp'
+  $log_port        = '601'
+  $log_serv        = '192.168.56.10'
+  $user_host       = 'localhost'
+  $java_mode       = 'jdk'
+  $jenkins_version = '6.7.2'
+
+  # set of apps needs to be logged
+  $jenkins = {
+    log_name => '/var/log/jenkins/*.log',
+    app_name => 'jenkins',
+    severity => 'info',
+  }
+  $apps   = [$jenkins]
+
+  class { 'java8':
+    java_se      => $java_mode,
+  }
   include jenkins
+
+  base::ssh_user { $ssh_user:
+    ssh_user     => $ssh_user,
+    ssh_group    => $ssh_group,
+    ssh_password => $ssh_password,
+  }
+  firewall::openport {'jenkins':
+    dports       => $dports,
+  }
+  rsyslog::client { 'app' :
```

Comment:
is shouldn't be "app" it should be "jenkins"

I bet log_proto and log_port are the same across all roles. Use defaults!